lib/meilisearch/index.rb

```diff
@@ -74,16 +74,11 @@ def documents(options = {})
     def add_documents(documents, primary_key = nil)
       documents = [documents] if documents.is_a?(Hash)
       http_post "/indexes/#{@uid}/documents", documents, { primaryKey: primary_key }.compact
-    end
```

Comment:
The code is still breaking @Smartmind12.

We need one method and two handlers inside of it based on a parameter from the arguments list.

```
   def add_documents(documents, primary_key = nil, wait: false)
      if wait
        # add documents that will have to wait until the end of the request
      else
        # add documents that will NOT have to wait until the end of the request
      end
    end
```